src/stylesheets/base/_typography.scss

```diff
@@ -29,6 +29,18 @@
     letter-spacing: none;
   }
 }
+
+.a-sup {
+  @include text($primary-font, normal, 1rem, $color-orange-red);
+
+  &:hover {
+    font-weight: bold;
+  }
```

Comment:
changing the font-weight on hover is generally a bad idea. It makes your link take up more room than pre-hover and can give you some funny text flashing/wrapping/moving

Reply:
ok. resolved in PR #4